### ISSUE TITLE
Training issue with noise_std

### ISSUE THREAD

**Issue by qilinli** [geoelements/gns#26]
Hey guys,

Many thanks for sharing a Pytorch version of GNS. It makes life much easier, compared to using the official tf version.

I'm currently using this code to train on my own dataset and one strange thing is that the training loss (one-step prediction) degrade quicker with higher noise_std and does not converge with noise_std=0, which is weird. My understanding is that this should only happen to the rollout prediction where more noise means more stable prediction.

Do you have similar observations or any clue on this?

**Comment by arjun-mani** [comment]
Some thoughts here based on my experience -- this might have to do with normalization (value of noise_std affects magnitude of targets). However, it should still converge with noise_std=0, that's unusual.
